### PR TITLE
Added a sponsor section to website

### PR DESCRIPTION
**Pull request by crxm**
I have added a sponsor section to the website as requested in the following issue:

https://github.com/zen-audio-player/zen-audio-player.github.io/issues/72

Images should be cropped to remove white space, and resized to a width of 400px, before stacking vertically.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -88,6 +88,10 @@
         <p>Created by <a href="https://twitter.com/_Shakeel" target="_blank">@_Shakeel</a></p>
         <p>Source available on <a href="https://github.com/zen-audio-player/zen-audio-player.github.io" target="_blank">GitHub</a></p>
         <p><a href="https://chrome.google.com/webstore/detail/zen-youtube-audio-player/jlkomkpeedajclllhhfkloddbihmcjlm" target="_blank">Get the Chrome extension</a> - (<a href="https://github.com/zen-audio-player/extension-chrome" target="_blank">source</a>)</p>
+        <!-- Sponsor links -->
+		<br/>
+        <p><h3>Sponsored by:</h3></p>
+        <p><a href="https://www.browserstack.com/"><img src="img/browserstack_logo.png" height="86" width="400"></a></p>
```
Comment (shakeelmohamed) [inline index.html]: Can we get some alt text on the image?

File: index.html [hunk 2]
```diff
@@ -88,6 +88,10 @@
         <p>Created by <a href="https://twitter.com/_Shakeel" target="_blank">@_Shakeel</a></p>
         <p>Source available on <a href="https://github.com/zen-audio-player/zen-audio-player.github.io" target="_blank">GitHub</a></p>
         <p><a href="https://chrome.google.com/webstore/detail/zen-youtube-audio-player/jlkomkpeedajclllhhfkloddbihmcjlm" target="_blank">Get the Chrome extension</a> - (<a href="https://github.com/zen-audio-player/extension-chrome" target="_blank">source</a>)</p>
+        <!-- Sponsor links -->
+		<br/>
```
Comment (shakeelmohamed) [inline index.html]: nit: tab indents

File: index.html (hunk 1 above)
Comment (shakeelmohamed) [inline index.html]: Can we add `target="_blank"` to the link?

**Comment by shakeelmohamed** [comment]
Awesome @crxm! Just left some nit comments

File: index.html [hunk 3]
```diff
@@ -88,6 +88,10 @@
         <p>Created by <a href="https://twitter.com/_Shakeel" target="_blank">@_Shakeel</a></p>
         <p>Source available on <a href="https://github.com/zen-audio-player/zen-audio-player.github.io" target="_blank">GitHub</a></p>
         <p><a href="https://chrome.google.com/webstore/detail/zen-youtube-audio-player/jlkomkpeedajclllhhfkloddbihmcjlm" target="_blank">Get the Chrome extension</a> - (<a href="https://github.com/zen-audio-player/extension-chrome" target="_blank">source</a>)</p>
+        <!-- Sponsor links -->
+	<br>
+        <p><h3>Sponsored by:</h3></p>
+        <p><a href="https://www.browserstack.com/" target="_blank"><img src="img/browserstack_logo.png" alt="Browserstack logo"></a></p>
```
Comment (shakeelmohamed) [inline index.html]: Can we fix the indents here? :smile:
Line 91 has a tab indent as well, thanks!

Comment (crxm) [inline index.html]: Sorry about that. I'm new to the GitHub editor, and I think I misunderstood you the first time. How does it look now?

Comment (shakeelmohamed) [inline index.html]: A+! This is perfect, thank you so much for doing this :trophy:

Comment (shakeelmohamed) [inline index.html]: @crxm there's also #77 & #78 if you're up for it

**Pull request merged**